common/src/main/java/org/wso2/testgrid/common/util/FileUtil.java

```diff
@@ -48,4 +59,42 @@
             return new Yaml().loadAs(fileInputStream, type);
         }
     }
+
+    /**
+     * This method returns the list of files in a given directory after filtering a given glob pattern.
+     * @param directory the directory in which the files should be look-up
+     * @param glob the glob pattern to be filtered
+     * @return list of files
+     * @throws TestGridException thrown when an error occurred while accessing files on directory.
+     */
+    public static List<String> getFilesOnDirectory(String directory, String glob) throws TestGridException {
+        List<String> files = new ArrayList<>();
+        try {
+            DirectoryStream<Path> stream = Files.newDirectoryStream(Paths.get(directory), glob);
```

Comment:
This may lead to a resource leak. We should use try-with-resources!
https://docs.oracle.com/javase/7/docs/api/java/nio/file/DirectoryStream.html